spacecmd/src/spacecmd/system.py

```diff
@@ -4563,6 +4563,9 @@ def do_system_bootstrap(self, args):
         if answer in ['y', 'Y']:
             options.saltssh = True
 
+    if isinstance(options.port, str) and options.port.isnumeric():
+        options.port = int(options.port)
+
```

Comment:
Should we exit early when `options.port` is not an `int()`?

Reply:
True, implemented.